### PR TITLE
Added hover background and click to sidebar nav

### PR DESCRIPTION
**Pull request by jninnes**
Supersedes #1317

This commit fixes the merge conflicts in #1317 and cleans up the code by:
- Using pure css for the background highlight
- Removing duplicate link code from the task template
- Addresses the style concerns of @kevinburke

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
@jninnes 
Thanks a lot for taking an action!

@tfennelly WDYT? There is commit authorship loss, but it's up to you.

**Comment by jninnes** [comment]
@oleg-nenashev While there's no visual change from @tfennelly , the implementation is different, so I updated authors. Happy to revert that if required.

**Comment by oleg-nenashev** [comment]
@jninnes Agreed, the implementation is different. No need to change authorship.

The change looks good to me, :+1:

**Comment by oleg-nenashev** [comment]
@recena WDYT, is it something recoverable?

**Comment by daniel-beck** [comment]
Could be interesting to go for consistency with the new management links. Assuming it can be done well.

Not all links are tasks, so there might be a mess of inconsistency. Unfortunately, no PR build due to merge conflicts.

**Comment by recena** [comment]
I could review that but there are no screenshots, conflicts, etc...

File: war/src/main/webapp/css/style.css [hunk 1]
```diff
@@ -28,12 +28,33 @@
   margin-bottom: 20px;
 }
 
+
```
Comment (alecharp) [inline war/src/main/webapp/css/style.css]: I don't think this is necessary

File: war/src/main/webapp/css/style.css [hunk 2]
```diff
@@ -393,7 +414,7 @@ pre.console {
 }
 
 #jenkins .task-link:hover {
-    text-decoration: underline;
+    text-decoration: none;
```
Comment (alecharp) [inline war/src/main/webapp/css/style.css]: I'm wondering if this is want we want as it will affect the visualization of the link

**Review by alecharp — COMMENTED** [review]
I would be great to have an before / after screenshot with this PR. 
Overall it looks good, I'm just concerned about the link not having an underline when pointer is over it.

**Comment by daniel-beck** [comment]
@jninnes Are you still interested in finishing this PR?

**Comment by daniel-beck** [comment]
Unfortunately we didn't properly handle it when it was new, and now there seems to be no interest anymore in getting this in in its current form. For general housekeeping, I'm closing this.

**Pull request closed without merging**